package.xml

```diff
@@ -58,6 +60,8 @@
   <run_depend>tf2_eigen</run_depend>
   <run_depend>tf2_geometry_msgs</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend>control_msgs</run_depend>
+  <run_depend>moveit_servo</run_depend>
```

Comment:
Nit: alphabetize. Sorry, again.

Can't you combine all of these `build_depend` and `run_depend` to just `depend`?

Reply:
I tried to lump under depend, and the build failed, saying it did not like the 'depend' tag

Reply:
Try setting the [package format](https://www.ros.org/reps/rep-0149.html) to `3` at the top of the file

`<package format="3">`

Reply:
That did the trick, thanks!